### DISCUSSION TITLE
api.search output is repeated

### DISCUSSION THREAD

**Discussion by qlofly** [tweepy/tweepy#1507] [Questions] [answered]
Hey. Why is it that when I do api.search, I get the same answer many times? And sometimes, instead of tweets, it redirects me to some sites somehow related to the word in the search

**Comment by Harmon758** [comment]
I'm not sure what you mean by "the same answer many times".
The same query will return the same Tweets if there haven't been any new Tweets that match the query since you last searched.

I'm also not sure what you mean by "redirects" you. [`API.search`](https://tweepy.readthedocs.io/en/latest/api.html#API.search) returns a list of Status/[Tweet objects](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/tweet).

**Comment by qlofly** [comment]
I mean, the same information is displayed multiple times. Example in the photo. Information is repeated several times: 
https://ibb.co/TYM3m3V
https://ibb.co/D9w3p4F
About the redirect. It seems to me like a link to a tweet, but the end result is some kind of site. If it's not a link to a tweet, then what?  - https://ibb.co/SVssQQb

**Comment by Harmon758** [comment, marked as answer]
Without the actual context, it's difficult to tell what these screenshots are showing.

The first screenshot seems to be the `_json` attribute of a User object, likely from the `author` attribute of a Status object.
The second screenshot is likely part of the `_json` attribute of a Status object, showing the `user` attribute.

The third screenshot is part of a user's description, which from [the profile itself](https://twitter.com/frugalmaniac), you can see contains the link in question, which is not to a Tweet.

**Comment by qlofly** [comment]
I think I get it. How can I customize the amount of information displayed?

**Comment by Harmon758** [comment]
How are you displaying it?

**Comment by qlofly** [comment]
For example, the Status part displays the following information: created_at, id, id_str, text, truncated, entities, urls, url, expanded_url and so on. How can I select only certain attributes to display?

**Comment by Harmon758** [comment]
Again, how are you displaying it?
You can access each attribute directly and pass it to whatever you use to display it.
